browser/.beforeprettier

```diff
@@ -93,6 +93,7 @@
 /src/core/Util.js
 /src/docstate.js
 /src/app/DocEvents.ts
+/src/app/GraphicSelectionMiddleware.ts
```

Comment:
please DO NOT add new code to the breforeprettier
this is only for OLD code we don't want to format